neurodamus/connection_manager.py

```diff
@@ -638,6 +638,7 @@ def _iterate_conn_params(self, src_target, dst_target, gids=None, show_progress=
             gids: Use given gids, instead of the circuit target cells. Default: None
             show_progress: Display a progress bar as tgids are processed
         """
+        crash_test_mode = SimConfig.cli_options.crash_test
```

Comment:
Could this be a dedicated `ConnectionManager`, inheriting from this class and overwriting some functionality instead?

Reply:
We don't need any change in the `ConnectionManager` anymore, I can even move this flag to the reader. I did have that thought for the `SonataReader` though, where at some point changes were significant.
In the current latest implementation I reduced the changes to a couple of conditional blocks. Maybe that's acceptable